### PR TITLE
add parameter to skip setting environment

### PR DESCRIPTION
**Pull request by j00p34**
I know this is a module to add values from vault to the environment, but I need a synchronous module to read from vault into normal variables your code is perfect for this. This requires a very small change so please consider my pull request

### CODE REVIEW

**Comment by devTristan** [comment]
Hey! Great idea, thanks for contributing!

A few things before this gets merged.

Looks like you've got an error on line 118, I think you meant `options.skipEnv` as the `skipEnv` variable isn't defined.

I'm not sure if `skipEnv` is really the best name for the option. In other libraries such as shelljs, if you want to pull the library into a global variable, you do something like `require('shelljs/global')`. Since this library is global by default, what about `local`? That'd be `options.local`, and `require('vault-env/local')` and so on.

Finally, as this change might be useful to others, it'd be a good idea to test and document it.

The tests are written [here](https://github.com/faradayio/vault-env-js/blob/master/test/index.js) and use [tape](https://www.npmjs.com/package/tape). All of the tests so far assume global behavior and check process.env for results. However, it shouldn't be too tricky to make one that doesn't. Let me know if you need help with this.

Finally, the readme is [here](https://github.com/faradayio/vault-env-js/blob/master/README.md), and a little blurb about the option wouldn't go amiss.

Let me know if I can help. Looking forward to merging :)

**Comment by j00p34** [comment]
Thank you for your quick reply. I'm new to nodejs but I think I'll be able
to make those changes (which I totally agree about by the way). I'll look
in to that after the weekend 😀

Op 20 jan. 2017 21:06 schreef "Tristan Davies" <notifications@github.com>:

**Comment by j00p34** [comment]
@devTristan can you please review my updated pull request. I don't know if this is the proper way of doing things in node but it works for me now. I've added tests for one and two local variables. Other cases are tested by the existing tests in my opinion.

File: README.md [hunk 1]
```diff
@@ -57,3 +57,12 @@ vaultEnv.on('DATABASE_URL', function (newDB, oldDB) {
   console.log('DATABASE_URL has changed to ' + newDB + ' from ' + oldDB)
 })
 ```
+Require `vault-env/local` and vault-env will not set your environment
+your variables will only be exported by the module as regular variables
+
+```js
+var secret = require('./vault-env/local')
```
Comment (devTristan) [inline README.md]: you want `require('vault-env/local')` here, without the `./`

**Comment by j00p34** [comment]
Done. And yes I tested the tests :-)

**Comment by devTristan** [comment]
Released as 4.2.0, thanks for contributing!

**Comment by j00p34** [comment]
Thank you for your time.

**Pull request merged**